src/Evenement/EventEmitterTrait.php

```diff
@@ -59,7 +59,7 @@ public function listeners($event)
         return isset($this->listeners[$event]) ? $this->listeners[$event] : [];
     }
 
-    public function emit($event, array $arguments = [])
+    public function emit($event, ...$arguments)
     {
         foreach ($this->listeners($event) as $listener) {
             call_user_func_array($listener, $arguments);
```

Comment:
Can't this `call_user_func_array` call be replaced by `$listener(...$arguments)`?